examples/material_gallery/lib/gallery/app.dart

```diff
@@ -3,34 +3,32 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
 
 import 'home.dart';
 
 class GalleryApp extends StatefulWidget {
   GalleryApp({ Key key }) : super(key: key);
 
-  static GalleryAppState of(BuildContext context) => context.ancestorStateOfType(const TypeMatcher<GalleryAppState>());
-
   @override
   GalleryAppState createState() => new GalleryAppState();
 }
 
 class GalleryAppState extends State<GalleryApp> {
   bool _lightTheme = true;
```

Comment:
The name of this variables sounds like it's value should be a Theme. Maybe `useLightTheme`?